ui/Screen/Project.svelte

```diff
@@ -45,10 +45,7 @@
   };
 
   onMount(() => {
-    localPeer.projectEvents.subscribe(event => {
-      if (!event) {
-        return;
-      }
+    localPeer.projectEvents.onValue(event => {
       if (event.urn === urn) {
         refresh();
       }
```

Comment:
I'm still unclear why we need to be reading events directly in UI components. Why can't this screen render content from a store directly (reactively), without having to know about how that store is updated? There's nothing special about this component, it should just render the state of the store :thinking:

Reply:
You’re correct. This code should be moved to `ui/src/screen/project.ts`. As mentioned in #1800 I deliberately excluded this change to focus on the improvements to the event filtering.

Reply:
Ah gotcha. Then we're on the same page :+1: